apollo-compiler/src/test/graphql/com/example/input_object_type/TestQueryExpected.java

```diff
@@ -0,0 +1,86 @@
+package com.example.input_object_type;
+
+import com.apollostack.api.Query;
+import java.lang.Override;
+import java.lang.String;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public final class TestQuery implements Query<TestQuery.Variables> {
```

Comment:
Should we be calling these mutations instead? We should use `operationType` field in the IR.

Not sure if it's worth renaming the Query interfaces to a `GraphQLOperation` interface. Then, having `GraphQLQuery` and `GraphQLMutation` extend those.
It's mainly for clarity since there's no real difference between queries and mutations other than the naming.